### ISSUE TITLE
Add ability to mark embedded document as optional

### ISSUE THREAD

**Issue by kop** [yii2tech/embedded#8]
`yii2tech\embedded\Validator` is validating all the embedded models by default and returns errors if child models are failed to validate. However, sometimes there are no need to validate such models - the developer may simply want to not create them.

Is it possible to add `optional` setting for `yii2tech\embedded\Validator` to let it know that embedded model may be empty?

**Comment by klimov-paul** [comment]
You should resolve this using standard Yii mechanisms for the particular field validation skip.
You may use the validation scenarios or specify direct list of attributes to be validated at `Model::validate()`.
The simpliest solution for you will be:

``` php
$model->validate($model->attributes()); // validate only attributes, skip all embedded
```

You already have all necessary tools.

**Comment by klimov-paul** [comment]
Reconsidered: it make sense to add option, which allows skipping validation in case embedded model has not been initialized yet.

**Comment by klimov-paul** [comment]
Resolved by commit 98e1ed1608f4b55e329f3f6e14fc6f865529e6d8
